### ISSUE TITLE
A11y_WINFORMS_CommonControls1_ButtonSection_Keyboard : Keyboard focus order is incorrect inside Button section.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/winforms#7336]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### .NET version
.NET Framework Version 4.8.04161
Operating System: Windows 11 Enterprise 21H2

### Did it work in .NET Framework?

No

### Issue description

GitHubTags:#A11yMAS;#A11yTCS;#.NET Core[WINFORMS]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG2.4.3;#DesktopApp;#Win11;#Benchmark;#Keyboard

### **Actual:**
Focus order is incorrect inside button section, as the keyboard focus is first moving to 'LargeIcon' instead of moving to 'List'.

### **Expected:**
Focus order should be correct inside button section, as the keyboard focus should first move to 'List' instead of moving to  'LargeIcon' and it should move in sequence.

### User Impact:

Keyboard users will face difficulties while navigating inside button section.

### Steps to reproduce

1. Open Core testing solution in VS Enterprise 2022 Int Preview.
2. Right click on cs.proj file and click on Build-> Build Solution.
3. Press enter on the application.
4. Main Form screen gets open
5. Click on "Common_Controls1:Testing the controls under the Common_controls" Tab.
6. Tab to Button section and navigate using keyboard Tab key.
7. Observe that Keyboard focus order is incorrect inside Button section.

### Attachment:

https://user-images.githubusercontent.com/90901519/175499271-23096fac-e9c0-42a7-8abf-1a01bbea9cfc.MP4

**Comment by RussKie** [comment]
Thank you for the submission. Unfortinately, I'm having troubles understanding the natute of the issue. Could you please provide an animaton showing the issue?

**Comment by merriemcgaw** [comment]
@PoojaNamde we need a video or a gif illustrating what specifically is wrong with the tab order. Please attach one, as there is not enough information for us to proceed with anything.

**Comment by PoojaNamde** [comment]
@merriemcgaw @RussKie 

Added attachment describing the issue.

**Comment by merriemcgaw** [comment]
@PoojaNamde thank you so much!
@Olina-Zhang this looks like a simple tab order issue. Can you update the sample app to have the correct tab order inside the sample and provide the update app to @PoojaNamde ?

**Comment by Amy-Li03** [comment]
@merriemcgaw @PoojaNamde have updated the sample app, you can get the new app from here:
[Accessibility_Core_7.0_App.zip](https://github.com/dotnet/winforms/files/8989190/Accessibility_Core_7.0_App.zip)

**Comment by merriemcgaw** [comment]
We have to keep these open until @PoojaNamde closes with their verification. @poojaNamde please let us know when you're able to do this verification.

**Comment by PoojaNamde** [comment]
Verified this issue in the latest environment. Issue is fixed hence closing the bug.

**Comment by MelonWang1** [comment]
Verified on .NET 7.0.100-preview.7.22370.3 TP build, issue is fixed, screenshot as below:
![7336](https://user-images.githubusercontent.com/94418985/180748393-01a3f4ef-bf9a-4063-ade3-eef0deb1cb1f.gif)
